types/index.d.ts

```diff
@@ -0,0 +1,90 @@
+// Type definitions for Chalk
+// Definitions by: Thomas Sauer <https://github.com/t-sauer>
+
+export const enum Level {
+	None = 0,
+	Basic = 1,
+	Extended = 2,
+	TrueColor = 3
+}
+
+export interface ChalkOptions {
+	enabled?: boolean;
+	level?: Level;
+}
+
+export interface Chalk {
+	new (options: ChalkOptions): Chalk;
```

Comment:
Shouldn't this be `options?: ChalkOptions`? I don't know enough about TS defs, but we don't require an object be passed.